doc/man7/openssl_user_macros.pod.in

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+openssl_user_macros, OPENSSL_API_COMPAT - User defined macros
+
+=head1 DESCRIPTION
+
+User defined macros allow the programmer to control certain aspects of
+what is exposed by the OpenSSL headers.
+
+B<NOTE:> to be effective, a user defined macro I<must be defined
+before including any header file that depends on it>, either in the
+compilation command (C<cc -DMACRO=value>) or by defining the macro in
+source before including any headers.
+
+Other manual pages may refer to this page when declarations depend on
+user defined macros.
```

Comment:
Aren't you stating the obvious here?

Reply:
I think of it of a declaration of what to expect in other pages (and btw, I will change other pages accordingly, in another PR)...  it may seem obvious, but I believe it clarifies to readers that there should be a mention in other pages.